json2xcom.cpp

```diff
@@ -686,6 +686,12 @@ saved_game build_save(const Json& json)
 
 void usage(const char * name)
 {
+	printf("json2xcom, a part of xcomsave, a save game editor for xcom games\n");
```

Comment:
I'm happy to have the contributors list somewhere (like a contributors file or in the README) but I think this is too long to have in the usage. It'll get way too spammy if there are many more contributors.

Reply:
For now I'll put it in the readme, ok?